create_infotexts.py

```diff
@@ -233,39 +232,44 @@ def create_commons_filename(self, item):
         :param item: dictionary conatining metadata for one image
         :return: None (populates self.filename)
         """
-        fname_str = ""
-        fname_str += re.sub(" Svenska Cypernexpeditionen\.?", "", item["Beskrivning"])
-        if not fname_str.endswith(" "):
-            fname_str += " "
+        fname_desc = ""
+        fname_desc += re.sub(" Svenska Cypernexpeditionen\.?", "", item["Beskrivning"])
+        if not fname_desc.endswith("."):
+            fname_desc += "."
 
         # Enrich with keywords
+        keywords_to_append = []
         if self.data["keyword_list"]:
             for keyword in self.data["keyword_list"]:
                 if keyword not in item["Beskrivning"]:
-                    fname_str += keyword + " "
+                    keywords_to_append.append(keyword)
+
+        for kw in keywords_to_append:
+            fname_desc += kw + ", "
+        fname_desc.rstrip(", ")
 
         # Enrich with regional data
         if not item["Ort, foto"] in item["Beskrivning"]:
-            fname_str += item["Ort, foto"] + " "
+            fname_desc += item["Ort, foto"] + ", "
```

Comment:
You need an initial space to separate it from the keyword list. I would probably do all of these as `fname_desc += " {},".format(item["..."])` and add a
```python
if  fname_desc.endswith(","):
    fname_desc = "{}.".format(fname_desc.strip(","))
```
To replace any trailing comma by a full stop